SQL_DDL_HW

dncsilla opened the pull request.

dncsilla committed:
SQL_DDL_HW

glybinahanna reviewed (changes requested):
Really great job! Thank you. Please see comments:
- script to drop database fails because such database doesn't exist. No script to create database.
- ADD COLUMN record_ts column scripts are not rerunable.

**Not approved, please make changes based on the feedback.**

dncsilla committed:
Update SQL_DDL_Csilla_Dan_HW.sql

Corrected the alter table queries and the drop table (I left the wrong one in the final code, instead of CREATE database), based on the review comments. Thank you!

dncsilla commented:
Hi Hanna, Thank you very much for your comments ! 
Corrected the alter table queries and the drop table (I left the wrong one in the final code, instead of CREATE database), based on the review comments. Thank you!

glybinahanna reviewed (approved):
Great job! **Approved. Please merge.**
